tools/m1_utils/update_static.sh

```diff
@@ -87,8 +87,22 @@ patch() {
              # We should also provide a way to use the secret "linker flag" to
              # allow the link to work
              echo "warning: falling back to input for $(basename $OF)"
+
+             # Xcode 13.1-13.2.1 workaround ( no issue for 12.5.1 ) Strip iOS
+             # device outputs file to mitigate LLDB default SDK selection from
+             # the DWARF entry - DW_AT_APPLE_sdk. No harm since many third party
+             # dist binaries ship with release + DSYM. Without this, Swift
+             # debugging is broken. To fix without stripping, implement a macho
+             # rewriter that removes or updates DW_AT_APPLE_sdk. Upstream LLVM
+             # has redesigned the logic and it's fixed there once devs have that
+             # patch
+             strip -S $OF
```

Comment:
🤯

Reply:
am I right that one day we can revert this patch once upstream LLVM is fixed?